source/includes/reference/_profiles.md

```diff
@@ -327,3 +327,354 @@ issuerCountry            | Issued by country code. For example "US"    | Text
+                 "dateOfBirth": "1982-05-20",
+                 "countryOfResidenceIso3Code": "usa"
+             },
+             {
+                 "firstName": "Jane",
+                 "lastName": "Doe",
+                 "dateOfBirth": "1981-12-07",
+                 "countryOfResidenceIso3Code": "usa"
+             }
+        ]'
+```
+
+> Example Response:
+
+```json
+[
+    {
+        "id": <director id>,
+        "firstName": "John",
+        "lastName": "Doe",
+        "dateOfBirth": "1982-05-20",
+        "countryOfResidenceIso3Code": "usa"
+    },
+    {
+        "id": <director id>,
+        "firstName": "Jane",
+        "lastName": "Doe",
+        "dateOfBirth": "1981-12-07",
+        "countryOfResidenceIso3Code": "usa"
+    },
+    {
+        "id": <director id>,
+        "firstName": "Oliver",
+        "lastName": "Wilson",
+        "dateOfBirth": "2017-12-31",
+        "countryOfResidenceIso3Code": "gbr"
+    }
+]
+```
+Adds new directors to the business profile.
+Returns the list of all directors associated with the business profile.
+
+### Request (Business)
+**`POST https://api.sandbox.transferwise.tech/v1/profiles/{profileId}/directors`**
+
+
+## Replace business directors
+
+> Example Request:
+
+```shell
+curl -X PUT https://api.sandbox.transferwise.tech/v1/profiles/{profileId}/directors \
+     -H "Authorization: Bearer <your api token>" \
+     -H "Content-Type: application/json" \
+     -d '[
+             {
+                 "firstName": "John",
+                 "lastName": "Doe",
+                 "dateOfBirth": "1982-05-20",
+                 "countryOfResidenceIso3Code": "usa"
+             },
+             {
+                 "firstName": "Jane",
+                 "lastName": "Doe",
+                 "dateOfBirth": "1981-12-07",
+                 "countryOfResidenceIso3Code": "usa"
+             }
+        ]'
+```
+
+> Example Response:
+
+```json
+[
+    {
+        "id": <director id>,
+        "firstName": "John",
+        "lastName": "Doe",
+        "dateOfBirth": "1982-05-20",
+        "countryOfResidenceIso3Code": "usa"
+    },
+    {
+        "id": <director id>,
+        "firstName": "Jane",
+        "lastName": "Doe",
+        "dateOfBirth": "1981-12-07",
+        "countryOfResidenceIso3Code": "usa"
+    }
+]
+```
+Overrides directors in the business profile.
+Returns the list of all directors associated with the business profile.
+
+### Request (Business)
+**`PUT https://api.sandbox.transferwise.tech/v1/profiles/{profileId}/directors`**
+
+
+## Get business ultimate beneficial owners
+
+> Example Request:
+
+```shell
+curl -X GET https://api.sandbox.transferwise.tech/v1/profiles/{profileId}/ubos \
+     -H "Authorization: Bearer <your api token>" 
+```
+
+> Example Response:
+
+```json
+[
+    {
+        "id": <shareholder id>,
+        "name": "John Doe",
+        "dateOfBirth": "1982-05-20",
+        "countryOfResidenceIso3Code": "usa",
+        "addressFirstLine": "87 West Central St",
```

Comment:
Use more obviously fake data here too please

Reply:
Changed address and post code to `123 Fake St` and `FK 12345`